src/codeu/chat/common/DerbyDatabaseInteractions.java

```diff
@@ -27,13 +27,15 @@
 	
 	public void addConversation(Conversation c) throws SQLException;
 	
-	public void updateConversation(Conversation c) throws SQLException;
+	public void updateConversationMessages(Conversation c) throws SQLException;
```

Comment:
Some javadoc comments explaining when these functions should be called would be helpful.